contrib/scripts/check_oid_ovfps.sh

```diff
@@ -0,0 +1,515 @@
+#!/bin/bash
+#
+#
+#  Copyright 2016 CUBRID Corporation
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+
+# declare read only variable
+declare -r -a key_titles=(
+    "max_num_ovf_page_a_key"            "Max_num_ovf_page_a_key"
+    "avg_num_ovf_page_per_key"          "Avg_num_ovf_page_per_key"
+    "num_ovf_page"                      "Num_ovf_page"
+    "avg_num_value_per_deduplicate_key" "Avg_num_value_per_deduplicate_key"
+)
+
+declare -r sort_option_page=1
+declare -r sort_option_owner=2
+declare -r sort_option_table=3
+declare -r sort_option_index=4
+declare -r -a  sort_title=(
+    "pages" "pages"       $sort_option_page
+    "owner" "owner-name"  $sort_option_owner
+    "table" "table-name"  $sort_option_table
+    "index" "index-name"  $sort_option_index
+)
+
+# global variable
+database=""
+user="-u dba"
+pass=""
+target_owner=""
+target_table=""
+connection_mode="-C"
+
+match_key="${key_titles[1]}"
+declare -i sort_field=${sort_title[2]}
+declare -i threshold_value=1000
+declare -i info_map_st_sz=6
+declare -a info_map  # array ( pages, owner-name, table-name, index-name, partition, invisible )
+declare IFS_backup=$IFS
+
+current_dir=""
+work_dir=""
+
+######################################################################
+# functions
+#
+function echo_stderr()
+{
+#   echo "$@" 1>&2;
+#   cat <<< "$@" 1>2&;
+    printf "%s\n" "$*" 1>&2;
+}
+
+function show_usage ()
+{
+	echo "Usage: $0 [OPTIONS] [database]"
+	echo " OPTIONS"
+	echo "  -C client-server mode execution, default"
+	echo "  -S standalone mode execution"
+	echo "  -u, --user=ARG      Set database user name(DBA or DBA group member); default dba"
+	echo "  -p, --password=ARG  Set password of databases user name; default NULL"
+	echo "  -o, --owner=ARG     Set target owner ID; default: NULL"
+	echo "  -c, --class=ARG     Set target class name; default: NULL"
+	echo "  -k, --key=ARG       Set the name of the key field to be checked; default: Max_num_ovf_page_a_key"
```

Comment:
It seems that it is difficult for the user to enter the key string. 
How about letting the user choice a number like this:
1 : Max_num_ovf_page_a_key (default)
2: Avg_num_ovf_page_per_key
3 : Num_ovf_page
4 : Avg_num_value_per_deduplicate_key

Reply:
OK, I will fix it.
thank you.